internal/pkg/dl/migration.go

```diff
@@ -74,7 +74,6 @@ func updateAgentMetadata(ctx context.Context, bulker bulk.Bulk, body []byte) (in
 	opts := []func(*esapi.UpdateByQueryRequest){
 		client.UpdateByQuery.WithBody(reader),
 		client.UpdateByQuery.WithContext(ctx),
-		client.UpdateByQuery.WithRefresh(true),
 		client.UpdateByQuery.WithConflicts("proceed"),
```

Comment:
had to remove 
```
client.UpdateByQuery.WithRefresh(true)
```
The migration is broken without this change. The fleet service token doesn't have permissions for refresh.

Reply:
Probably should comment it out with a warning rather than just delete.